examples/tv-casting-app/tv-casting-common/core/CommissionerDeclarationHandler.cpp

```diff
@@ -50,6 +51,25 @@ void CommissionerDeclarationHandler::OnCommissionerDeclarationMessage(
     chip::Server::GetInstance().GetCommissioningWindowManager().CloseCommissioningWindow();
     support::ChipDeviceEventHandler::SetUdcStatus(false);
 
+    // Flag to indicate when the CastingPlayer/Commissioner user has decided to exit the commissioning process.
+    if (cd.GetCancelPasscode())
+    {
+        ChipLogProgress(AppServer,
+                        "CommissionerDeclarationHandler::OnCommissionerDeclarationMessage(), Got CancelPasscode parameter, "
+                        "CastingPlayer/Commissioner user has decided to exit the commissioning attempt. Connection aborted.");
+        // Since the user has decided to exit the commissioning process on the CastingPlayer/Commissioner, we cancel the ongoing
+        // connection attempt without notifying the CastingPlayer/Commissioner.
+        if (auto sharedPtr = mTargetCastingPlayer.lock())
```

Comment:
Instead of referring directly to the private data member, use the static CastingPlayer::GetTargetCastingPlayer() method

Reply:
Ok.